pyfar/dsp/dsp.py

```diff
@@ -1498,3 +1498,124 @@ def decibel(signal, domain='freq', log_prefix=None, log_reference=1,
         return log_prefix * np.log10(np.abs(data) / log_reference), log_prefix
     else:
         return log_prefix * np.log10(np.abs(data) / log_reference)
+
+
+def average(signal, mode='time', axis=None, phase_copy=None,
```

Comment:
docstring and checks for axis are missing. We have to make sure that `np.max(axis) <= len(signal.cshape)`